extensions/proposals/WEBGL_webcodecs_video_frame/webgl_webcodecs_video_frame.js

```diff
@@ -4,6 +4,12 @@ Use of this source code is governed by an MIT-style license that can be
 found in the LICENSE.txt file.
 */
 
+let webgl_webcodecs_test_context_ = null;
+
+function setTestMode(webgl_webcodecs_test_context) {
+    webgl_webcodecs_test_context_ = webgl_webcodecs_test_context;
+}
+
 function requestWebGLVideoFrameHandler(canvas) {
```

Comment:
For future work: this function is very large and it's hard to read through it to see the values it returns. Please consider refactoring this in similar form to WebGLTestUtils (or use real classes) so the internal functions / methods can be separated out while keeping this function / method short.